source/reference/api/start.txt

```diff
@@ -141,10 +141,14 @@ Request Body Parameters
          cluster while migration is in progress, and unblocks writes right
          before ``canWrite`` is ``true``.
 
-       To reverse sync, the ``enableUserWriteBlocking`` field must be set
-       to ``"sourceAndDestination"``. To allow the source cluster 
-       to accept writes again, for example after running migration tests, 
-       run the following command: 
+       .. include:: /includes/fact-permissions-body.rst
```

Comment:
I know we talked about this briefly over Zoom, but looking at this now I'm not totally sure we need this include here? Since we have this content at the top of the page, it kind of makes sense to me that we use this space to get more specific as you've done with the added info for reverse sync, but maybe we don't need to repeat the include copy? What do you think?